### DISCUSSION TITLE
What is memory and disk usage of the server?

### DISCUSSION THREAD

**Discussion by ardabro** [janoside/btc-rpc-explorer#452] [Q&A]
Will I be able to run it together with bitcoin core on a tiny low-power x64, 4GB RAM machine with Debian ?

**Comment by janoside** [comment]
Memory usage will depend on usage, but disk usage is basically zero.

But, to your next question: Yes, this tool is specifically designed for low-power hardware. By default it runs in "slow device mode", meaning some of the more power-hungry features are disabled.

Please give it a try and fee free to report any issues that arise.
